skimage/graph/_graph.py

```diff
@@ -149,7 +149,7 @@ def central_pixel(graph, nodes=None, shape=None, partition_size=100):
 
     Parameters
     ----------
-    graph : scipy.sparse.csr_matrix
+    graph : scipy.sparse.csr_array
```

Comment:
```suggestion
    graph : scipy.sparse.csr_array or scipy.sparse.csr_matrix
```
Do we intend to be backward compatible here and continue supporting `csr_matrix`? In that case we need a test to ensure correctness, if not, I'd think that we need a deprecation.

Same for other places. I'm happy to work on this.